components/SearchBar/index.tsx

```diff
@@ -68,6 +82,9 @@ function Component() {
         toastId: 4,
       });
     }
+    if (zilAddr !== null) {
```

Comment:
better with a switch function (look into zilpay api)

Reply:
ZilPay is working on an update that will disconnect/ remove connections from api)